presto-hive-hadoop2/src/main/java/io/prestosql/plugin/hive/HiveHadoop2Plugin.java

```diff
@@ -23,6 +23,9 @@
     @Override
     public Iterable<ConnectorFactory> getConnectorFactories()
     {
-        return ImmutableList.of(new HiveConnectorFactory("hive-hadoop2"));
+        return ImmutableList.of(
+                new HiveConnectorFactory("hive"),
```

Comment:
We haven't done this kind of aliasing in the past, so I'm curious why not. Hopefully @electrum remembers.

Reply:
In general, this won't work, since you can't have multiple connectors with the same handle classes. For Hive, this works now that we use a separate class loader per connector instance. I'm not sure if this will cause other issues.